WordPress/Classes/Services/BlogService.m

```diff
@@ -744,6 +734,27 @@ - (NSTimeZone *)timeZoneForBlog:(Blog *)blog
     return timeZone;
 }
 
+- (void)updateBlog:(Blog *)blog settingsWithRemoteSettings:(RemoteBlogSettings *)remoteSettings
+{
+    blog.blogName = remoteSettings.name;
+    blog.blogTagline = remoteSettings.desc;
+    if (remoteSettings.defaultCategory) {
```

Comment:
Are we checking `if (remoteSettings.defaultCategory)` for any particular reason? (just wondering if the three checks are 100% required, or it breaks)